### PR TITLE
updtaed combo, combocontroller, vendor, vendorcontroller

### PR DESCRIPTION
**Pull request by jayharris14**

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Kev-Index) [inline .project]: What is this file?

File: QuickBytes/src/main/java/com/quickbytes/controller/ComboController.java [hunk 2]
```diff
@@ -1,13 +1,87 @@
...
+		
+		@GetMapping("/combos/vendorId/{vid}")
+		public List<Combo> getCombosByVendorId(@PathVariable("vid") Long vid){
+			List<Combo> list = comboRepository.getCombosByVendorId(vid);
+			return list;
+		}
+		
+		@GetMapping("/combo/{name}")
+		public Long getComboId(@PathVariable("name") String name) {
+			Optional<Combo> optional=comboRepository.findByname(name);
```
Comment (Kev-Index) [inline QuickBytes/src/main/java/com/quickbytes/controller/ComboController.java:66]: Can we change this and the other 3 occurrences to findByName?

File: QuickBytes/src/main/java/com/quickbytes/controller/ComboController.java [hunk 3]
```diff
@@ -1,13 +1,87 @@
-package com.quickbytes.controller;
+	package com.springboot.backend.controller;
```
Comment (Jlpowellv) [inline QuickBytes/src/main/java/com/quickbytes/controller/ComboController.java]: we should not rename packages, this will mess things up.

Comment (Jlpowellv) [reply QuickBytes/src/main/java/com/quickbytes/controller/ComboController.java]: actually It probably would be easier to edit your packages to fit to our package path, if you would not mind doing that

File: QuickBytes/src/main/java/com/quickbytes/model/Combo.java [hunk 4]
```diff
@@ -1,20 +1,74 @@
...
+@Table(name= "combos")
 public class Combo {
-
 	@Id
 	@GeneratedValue(strategy = GenerationType.AUTO)
 	private Long comboId;
 	
-	@OneToMany //fk
-	private List<RequestCombo> requestCombos;
-}
```
Comment (Jlpowellv) [inline QuickBytes/src/main/java/com/quickbytes/model/Combo.java:31]: add this to your code for all of the OneToMany calls, there are a few that are in the main branch

File: QuickBytes/src/main/java/com/quickbytes/controller/ComboController.java (hunk 3 above)
Comment (Kev-Index) [reply QuickBytes/src/main/java/com/quickbytes/controller/ComboController.java]: This error needs to be fixed in all files

File: QuickBytes/src/main/java/com/quickbytes/model/Vendor.java [hunk 5]
```diff
@@ -1,17 +1,47 @@
-package com.quickbytes.model;
+package com.springboot.backend.model;
```
Comment (Jlpowellv) [inline QuickBytes/src/main/java/com/quickbytes/model/Vendor.java:2]: please make constructors with fields and from superclass in both combo and vendor

File: QuickBytes/src/main/java/com/quickbytes/controller/UserController.java [hunk 6]
```diff
@@ -59,6 +60,11 @@ public void postUser(@RequestBody UserInfo user){
 			c.setUserId(user);
 			customerRepository.save(c);
 		}
+		if (user.getRole().equalsIgnoreCase("Vendor")) {
+			Vendor v=new Vendor();
+			v.setName((String)user.getUsername()+ "Please Change Name");
+			v.setBusinessId((long) 0);
```
Comment (Kev-Index) [inline QuickBytes/src/main/java/com/quickbytes/controller/UserController.java:77]: Save vendor with vendorRepository

File: QuickBytes/src/main/resources/application.properties [hunk 7]
```diff
@@ -1,5 +1,5 @@
 #tomcat (by defauly works on 8080 port)
-server.port=8989
+server.port=8389
```
Comment (Kev-Index) [inline QuickBytes/src/main/resources/application.properties:2]: Is there a reason for the change in server ports?

**Review by Kev-Index — APPROVED** [review]

**Pull request merged**